<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminUi</class>
 <widget class="QMainWindow" name="AdminUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="coursesBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>801</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListView" name="listViewCourses">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Courses</string>
     </property>
    </widget>
    <widget class="QPushButton" name="update_course_button">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>430</y>
       <width>150</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Course</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_new_course_button">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>430</y>
       <width>150</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Course</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_course_button">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>430</y>
       <width>150</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Course</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_close_button">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>2000</y>
       <width>110</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Save and Close</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>481</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Course ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_Id"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Department</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_department"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Schedule</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_schedule"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Pre-Requisits</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_prerequists"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Instructor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_instructor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_status"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Deadline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_deadline"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="studentsBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>801</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListView" name="listViewStudents">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Students</string>
     </property>
    </widget>
    <widget class="QPushButton" name="update_student_button">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>430</y>
       <width>150</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Student</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_new_student_button">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>430</y>
       <width>150</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Student</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_student_button">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>430</y>
       <width>150</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Student</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_close_button_2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>2000</y>
       <width>110</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Save and Close</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>481</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_name"/>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_email"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Student Id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_id"/>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_password"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Courses</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_course"/>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Academic Year</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_academic_year"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Completed Courses</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_completed_courses"/>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Events</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_events"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="eventsBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>801</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListView" name="listViewEvent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Events</string>
     </property>
    </widget>
    <widget class="QPushButton" name="update_event_button">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>430</y>
       <width>150</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Event</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_new_event_button">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>430</y>
       <width>150</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Event</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_event_button">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>430</y>
       <width>150</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Event</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_close_button_3">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>2000</y>
       <width>110</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Save and Close</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>481</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Event ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="event_id"/>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="event_title"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="event_description"/>
        </item>
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Speaker</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="event_speaker"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Schedule</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="event_schedule"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="coursesBtn">
       <property name="text">
        <string>Courses</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="studentsBtn">
       <property name="text">
        <string>Students</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="eventsBtn">
       <property name="text">
        <string>Events</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
